### PR TITLE
Storefront Cli

### PR DESCRIPTION
**Pull request by filrak**
PoC of Storefront CLI for scaffolding VS apps.

TODO:

- [x] Rename `create-storefront-app` to `vsf init`

TODO in next releases:
- Error handling
- `vsf upgrade`

### CODE REVIEW

File: src/modules/index.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ export const registerModules: VueStorefrontModule[] = [
   PaymentBackendMethods,
   PaymentCashOnDelivery,
   RawOutputExample,
-  AmpRenderer,
+  // AmpRenderer,
```
Comment (pkarw) [inline src/modules/index.ts]: Why this is commented out?

File: packages/cli/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## Installation
+
+``` 
+npm i -g @vue-storefront/cli 
+```
+## Commands
+
+After installing CLI you'll get access to following commands:
+- `create-storefront-app` - Use it to set up new VS project. The CLI will ask you a few questions and set up new, ready to work with Vue Storefront instance.
```
Comment (patzick) [inline packages/cli/README.md]: i've seen that you want another name, so needs to be changed here also

File: packages/cli/package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@vue-storefront/cli",
+  "version": "0.0.4",
```
Comment (patzick) [inline packages/cli/package.json]: shouldn't we keep versioning compatible for easier maintaining?
```suggestion
  "version": "1.10.0-rc.1",
```

File: packages/cli/package.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "init": "./index.js"
+  },
+  "bin": {
+    "create-storefront-app": "./bin/createStorefrontApp.js"
```
Comment (patzick) [inline packages/cli/package.json]: script is init, shouldn't here be the same?

File: .eslintignore [hunk 5]
```diff
@@ -1,2 +1,3 @@
 core/build/*.js
 node_modules
+packages/theme-capybara
```
Comment (patzick) [inline .eslintignore]: no need to add this

File: .gitignore [hunk 6]
```diff
@@ -29,6 +29,8 @@ core/resource/i18n/nl-NL.json
 core/resource/i18n/pl-PL.json
 core/resource/i18n/pt-BR.json
 core/resource/i18n/ru-RU.json
+_alpaca.scss
```
Comment (patzick) [inline .gitignore]: no need to add this

**Pull request merged**